frontend-react/src/components/Content/MarkdownRSLink.tsx

```diff
@@ -21,39 +21,8 @@ export type MarkdownComponentProps<
     children: React.ReactNode;
 };
 
-/**
- * Try to parse the href as a URL. If it throws, then it's not
- * an absolute href (aka is internal). If it parses, verify it is
- * from the cdc.gov domain (aka is internal).
- */
-export function isExternalUrl(href?: string) {
-    if (href === undefined) return false;
-    try {
-        // Browsers allow // shorthand in anchor urls but URL does not
-        const url = new URL(
-            href.replace(/^\/\//, `${window.location.protocol}//`)
-        );
-        return (
-            url.protocol.startsWith("http") &&
-            url.host !== "cdc.gov" &&
-            !url.host.endsWith(".cdc.gov")
-        );
-    } catch (e: any) {
-        return false;
-    }
-}
-
 export type MarkdownRSLinkProps = MarkdownComponentProps<"a">;
 
-export const MarkdownRSLink = ({ children, ...props }: MarkdownRSLinkProps) => {
-    let isExternal = props.href !== undefined;
-
-    if (props.href !== undefined) {
-        isExternal = isExternalUrl(props.href);
-    }
-
-    if (isExternal) {
-        return <USExtLink {...props}>{children}</USExtLink>;
-    }
-    return <USLink {...props}>{children}</USLink>;
+export const MarkdownRSLink = (props: MarkdownRSLinkProps) => {
```

Comment:
Can we just remove this component entirely since it's just an alias to USSmartLink now?